src/main/scala/nl/knaw/dans/easy/ingest/Main.scala

```diff
@@ -41,16 +41,16 @@ object Main {
   type PidDictionary = Map[ObjectName, Pid]
   type ConfigDictionary = Map[ObjectName, DOConfig]
 
-  case class DatastreamSpec(name: String, mime: String)
-  case class Relation(predicate: Predicate, objectName: ObjectName)
-  case class DOConfig(namespace: String, datastreams: List[DatastreamSpec], relations: List[Relation])
+  case class DatastreamSpec(contentFile: String = "", dsLocation: String = "", dsID: String = "", mimeType: String = "application/octet-stream", controlGroup: String = "M", checksumType: String = "", checksum: String = "")
+  case class Relation(predicate: Predicate, objectSDO: ObjectName = "", `object`: Pid = "")
+  case class DOConfig(namespace: String, label: String, ownerId: String, datastreams: List[DatastreamSpec], relations: List[Relation])
 
   def main(args: Array[String]) {
     // TODO: use Scallop for solid command line parsing
     if(args.length < 1) {
       println("Not enough arguments")
       System.exit(1)
-    }    
+    }
```

Comment:
main is getting quite large. It might be a good idea to delegate some more stuff to helper functions.